x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiBasicTable,
+  EuiText,
+  EuiTableFieldDataColumnType,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  FormattedMessage,
+  FormattedDate,
+  FormattedTime,
+  FormattedNumber,
+  FormattedRelative,
+} from '@kbn/i18n/react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { usePageId } from '../use_page_id';
+import {
+  selectIsLoading,
+  selectPageIndex,
+  selectPageSize,
+  selectPolicyItems,
+  selectTotal,
+} from '../../store/policy_list/selectors';
+import { usePolicyListSelector } from './policy_hooks';
+import { PolicyListAction } from '../../store/policy_list';
+import { PolicyData } from '../../types';
+import { TruncateText } from '../../components/truncate_text';
+
+interface TTableChangeCallbackArguments {
```

Comment:
Is `TT` on purpose or spelling?

Reply:
I kind of like that some types initial letter alludes the type. When I'm `import`'ing, I find it harder to determine what is executable code and what is a Type (_even with the fact that my IDE provides different icons_). But - I think this is anti-pattern nowadays right? to name your types starting with `I` (interface), `T` (type)?

I will rename it to remove the initial `T`.